File: challenges/2-idade-em-dias/javascript/codinghobbit/desafio2.js
const readline = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout,
});

readline.question("Insira idade em dias: ", (age) => {
  const ageAsNumber = Number(age);
  const ageInYears = parseInt(ageAsNumber / 365);
  const ageInMonths = parseInt((ageAsNumber % 365) / 30);
  const ageInDays = parseInt((ageAsNumber % 365) % 30);
  console.log(`
    ${ageInYears} ${ageInYears > 1 ? "anos" : "ano"}
    ${ageInMonths} ${ageInMonths > 1 ? "meses" : "mês"}
    ${ageInDays} ${ageInDays > 1 ? "dias" : "dia"}`);
  readline.close();
});

File: challenges/2-idade-em-dias/python/camiscf/desafio2.py
#idade em dias
idade = int(input())
anos = idade // 365
meses = (idade % 365) // 30
dias = (idade % 365) % 30
print('{} ano(s)'.format(anos))
print('{} mes(es)'.format(meses))
print('{} dia(s)'.format(dias))
